packages/frontend/src/redux/slices/nft/index.js

```diff
@@ -106,11 +106,6 @@ const nftSlice = createSlice({
                 const { contractName, tokens, accountId } = payload;
                 set(state, ['ownedTokens', 'byAccountId', accountId, 'byContractName', contractName, 'hasFetchedAllTokensForContract'], tokens.length < TOKENS_PER_PAGE);
                 update(state, ['ownedTokens', 'byAccountId', accountId, 'byContractName', contractName, 'tokens'], (n) => (n || []).concat(tokens));
-            },
-            clearState(state) {
```

Comment:
I think we should still have this reducer here, but listening to the account `CLEAR_ACCOUNT_STATE` action instead of having one defined here in-line; otherwise we won't clear the owned tokens state anymore when switching accounts.

Reply:
`onwedTokens` in NFT are keyed by `accountId` so we don't need to clear it :)

Reply:
:+1: roger that :D <3